DEBUGGING.md

```diff
@@ -0,0 +1,8 @@
+# DEBUGGING
```

Comment:
Maybe you could make a note about this somewhere else? I don't think this should be in the repo unless we have a really good reason for it.